repos/annotation-studio/src/OmekaItemExpander.php

```diff
@@ -67,12 +69,21 @@ public static function expandDocument($document, callable $resolveUrl = null, Tr
                     // Can't translate URI.
                     $jsonLd[$key] = $field;
                 } else {
-                    $jsonLd[$key] = array_map(
-                        function ($data) use ($translator) {
-                            return self::toJsonValue($data, $translator);
-                        },
-                        $field
-                    );
+                    $locale = $translator->getDelegatedTranslator()->getLocale();
+                    $fallback = $first;
+
+                    foreach ($field as $singleField) {
+                        if ($singleField->lang() === $locale) {
```

Comment:
`Locale::filterMatches(...)` might be a bit more "correct", though don't think it'll change the behaviour of this comparison.

Reply:
Oh, I see you doing something maybe along those lines in langMatches.

Reply:
Yeh, this will first check if its an exact match, then if not fallback to a fuzzy match (`en` vs `en_US`)